gui.py

```
@@ -0,0 +1,550 @@
+            rl.addWidget(self._btn("Browse…", pick, 80))
+        elif browse_file:
+            def pick(e=edit, ff=file_filter):
+                p, _ = QFileDialog.getOpenFileName(self, "Choose file", filter=ff)
+                if p: e.setText(p)
+            rl.addWidget(self._btn("Browse…", pick, 80))
+        rl.addStretch()
+        lay.addWidget(row)
+        return edit
+
+    def _hw_row(self, lay):
+        row = QWidget(); row.setStyleSheet(f"background: {CARD};")
```

Ruff (0.15.9)

[error] Multiple statements on one line (semicolon)

(E702)